content/en/docs/setup/install/helm/index.md

```diff
@@ -42,6 +42,14 @@ and then further customize the configuration for your specific needs.
 
 1. [Install a Helm client](https://github.com/helm/helm/blob/master/docs/install.md) with a version higher than 2.10.
 
+**Optional:** To enable SDS in Istio, upgrade your Kubernetes cluster to at least 1.13.
```

Comment:
Are these steps of the installation? Are these considerations the reader needs to make before installing? If they are, they should not be at the end of the procedure. Please include this information on the `Before you begin` section.
Please make sure that there is no duplication or repetition of the information. Do not add platform-specific information to the general installation flow, add it to the platform-specific instructions.

Reply:
I'll move it to the before you install section.